protos/dcs/coalition/v0/coalition.proto

```diff
@@ -22,7 +22,7 @@ service CoalitionService {
   rpc GetBullseye(GetBullseyeRequest) returns (GetBullseyeResponse) {}
 
   // https://wiki.hoggitworld.com/view/DCS_func_getPlayers
-  rpc GetPlayers(GetPlayersRequest) returns (GetPlayersResponse) {}
+  rpc GetPlayerUnits(GetPlayerUnitsRequest) returns (GetPlayerUnitsResponse) {}
```

Comment:
I've updated the method name on the RPC service to be inline with the behavior observations. Let me know if you agree with this.